### DISCUSSION TITLE
installing `@tailwindcss/oxide` returns rust files, and no index.js

### DISCUSSION THREAD

**Discussion by rawkakani** [tailwindlabs/tailwindcss#15254] [Help]
Hi I am trying to use `@tailwindcss/oxide` directly and when I install it from npm it returns rust files tried compiling it but, tailwindcss-core is not included 

is this how the npm package is supposed to come and how can I use it directly?

**Comment by christopher356** [comment]
@tailwindcss/oxide is a Rust-based implementation of Tailwind CSS for custom setups. To use Tailwind CSS in your JavaScript or TypeScript project, install it with its [ZYN Nicotine Pouches](https://www-zynrewards.com/) dependencies, initialize it, configure it, and then build your CSS using a tool like Webpack or Vite. For pre-built components, consider using frameworks like DaisyUI or Radix UI.

**Reply by rawkakani** [reply]
okay thank you for this, using postcss with taildind4 was giving errors so was trying a different approach
